Add readme to configurator (#15)
btcpayserver/btcpayserver-configurator: README.md

# Deploy BTCPay with Configurator

> BTCPay Server can easily be configured and deployed to a server using Configurator. This tool makes makes deployment simple by allowing users to select their BTCPay setup from the Configurator.

Reviewer: ```suggestion
BTCPay Server can easily be configured and deployed to a server using Configurator. This tool makes makes deployment simple by allowing users to initialize or modify their BTCPay setup from the Configurator.
```

> Configurator can only be enabled on a BTCPay Server admin account. Users may visit the server's Configurator url to deploy new BTCPay instances. 

Reviewer: ```suggestion
Configurator can only be used to modify the existing BTCPay Server on a BTCPay Server admin account. Other users may visit the server's Configurator url to deploy new BTCPay instances. 
```

Someone such as a BTCPay third-party host may provide a Configurator instance for their users. This can help transition users to a self-hosted solution when they are ready to stop using the third-party host's server.

> Another use case is for server admins who are deploying BTCPay Server's on behalf of clients or other users as part of a consulting business. Admins can easily export a Docker deployment script from Configurator selections or deploy the configuration immediately to a VPS or on premise server using SSH.

Reviewer: ```suggestion
Another use-case is for server admins who are deploying BTCPay Server's on behalf of clients or other users as part of a consulting business. Admins can easily export a Docker deployment script from Configurator selections or deploy the configuration immediately to a VPS or on-premise server using SSH.
```

# How to use Configurator

This deployment method assumes you already have an existing BTCPay Server [deployed](https://docs.btcpayserver.org/Deployment/) with the `opt-add-configurator` [environment variable added](https://docs.btcpayserver.org/FAQ/FAQ-Deployment#how-can-i-modify-or-deactivate-environment-variables) to your server and a target server where you would like to deploy a second BTCPay. On your current server, view your Configurator by navigating to: 

> **Server Settings > Services > Other external services > Configurator > Click See information**

Reviewer: We should mention that if you arent an admin, you go to /configurator

Step 1: Destination

Select an option to deploy using SSH now or generate a bash script for later deployment.

![Select Deployment](./docs/img/ConfiguratorStep1.png)

To configure and deploy a server now, provide your SSH credentials where you would like it deployed to. 

![Provide SSH Details](./docs/img/ConfiguratorStep1ssh.png)

> Note: The "Load Existing Settings" option will use the previous deployment's selections for faster configuration.

Reviewer: ```suggestion
Note: The "Load Existing Settings" option will use the previous deployment's selections for faster configuration if you are modifying an existing installation.
```

Someone such as a BTCPay third-party host may provide a Configurator instance for their users. This can help transition users to a self-hosted solution when they are ready to stop using the third-party host's server.

Another use-case is for server admins who are deploying BTCPay Server's on behalf of clients or other users as part of a consulting business. Admins can easily export a Docker deployment script from Configurator selections or deploy the configuration immediately to a VPS or on-premise server using SSH.

# How to set up Configurator

## Option 1: Add as an external service to BTCPay

If you already have an existing BTCPay Server [deployed](https://docs.btcpayserver.org/Deployment/) with the `opt-add-configurator` [environment variable added](https://docs.btcpayserver.org/FAQ/FAQ-Deployment#how-can-i-modify-or-deactivate-environment-variables), view your Configurator by navigating to: 

**Server Settings > Services > Other external services > Configurator > Click See information**

Once enabled, non-admins may also view the Configurator at: `yourbtcpaydomain.com/configurator`.

## Option 2: Build locally with Docker

> If you have Docker installed on your machine, you can open a terminal and the run the following command to build Configurator inside of a Docker container to use on your local machine:

Reviewer: ```suggestion
If you have Docker installed on your machine, you can open a terminal and the run the following command to run Configurator inside of a Docker container to use on your local machine:
```
